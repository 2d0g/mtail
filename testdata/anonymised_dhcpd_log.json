[
  {
    "Name": "request_total",
    "Program": "dhcpd.mtail",
    "Kind": 0,
    "Keys": [
      "command"
    ],
    "LabelValues": [
      {
        "Labels": [
          "balancing"
        ],
        "Value": {
          "Value": 9317,
          "Time": "0000-11-16T07:45:14Z"
        }
      },
      {
        "Labels": [
          "balanced"
        ],
        "Value": {
          "Value": 9317,
          "Time": "0000-11-16T07:45:14Z"
        }
      }
    ]
  },
  {
    "Name": "config_file_errors",
    "Program": "dhcpd.mtail",
    "Kind": 0
  },
  {
    "Name": "peer_disconnects",
    "Program": "dhcpd.mtail",
    "Kind": 0
  },
  {
    "Name": "dhcpdiscovers",
    "Program": "dhcpd.mtail",
    "Kind": 0,
    "Keys": [
      "mac"
    ]
  },
  {
    "Name": "bind_xid_mismatch",
    "Program": "dhcpd.mtail",
    "Kind": 0,
    "LabelValues": [
      {
        "Value": {
          "Value": 4088,
          "Time": "0000-11-16T07:46:05Z"
        }
      }
    ]
  },
  {
    "Name": "duplicate_lease",
    "Program": "dhcpd.mtail",
    "Kind": 0,
    "LabelValues": [
      {
        "Value": {
          "Value": 53,
          "Time": "0000-11-16T07:44:51Z"
        }
      }
    ]
  },
  {
    "Name": "bad_udp_checksum",
    "Program": "dhcpd.mtail",
    "Kind": 0
  },
  {
    "Name": "unknown_subnet",
    "Program": "dhcpd.mtail",
    "Kind": 0
  },
  {
    "Name": "dhcpdiscover_nofree",
    "Program": "dhcpd.mtail",
    "Kind": 0,
    "Keys": [
      "network"
    ]
  },
  {
    "Name": "unknown_lease",
    "Program": "dhcpd.mtail",
    "Kind": 0,
    "Keys": [
      "ip"
    ]
  },
  {
    "Name": "update_rejected",
    "Program": "dhcpd.mtail",
    "Kind": 0
  },
  {
    "Name": "failover_peer_timeout",
    "Program": "dhcpd.mtail",
    "Kind": 0
  },
  {
    "Name": "ip_already_in_use",
    "Program": "dhcpd.mtail",
    "Kind": 0
  },
  {
    "Name": "ip_abandoned",
    "Program": "dhcpd.mtail",
    "Kind": 0,
    "Keys": [
      "reason"
    ],
    "LabelValues": [
      {
        "Labels": [
          "pinged before offer"
        ],
        "Value": {
          "Value": 2,
          "Time": "0000-11-16T07:32:49Z"
        }
      }
    ]
  },
  {
    "Name": "invalid_state_transition",
    "Program": "dhcpd.mtail",
    "Kind": 0
  },
  {
    "Name": "negative_poolreq",
    "Program": "dhcpd.mtail",
    "Kind": 0,
    "Keys": [
      "pool"
    ]
  },
  {
    "Name": "lease_conflicts",
    "Program": "dhcpd.mtail",
    "Kind": 0
  }
]
