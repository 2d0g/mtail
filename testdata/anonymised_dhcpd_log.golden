counter request_total {command=balancing} 9317 0000-11-16T07:45:14Z
counter request_total {command=balanced} 9317 0000-11-16T07:45:14Z
counter config_file_errors
counter peer_disconnects
counter dhcpdiscovers {mac=}
counter bind_xid_mismatch 4088 0000-11-16T07:46:05Z
counter duplicate_lease 53 0000-11-16T07:44:51Z
counter bad_udp_checksum
counter unknown_subnet
counter dhcpdiscover_nofree {network=}
counter unknown_lease {ip=}
counter update_rejected
counter failover_peer_timeout
counter ip_already_in_use
counter ip_abandoned {reason=pinged before offer} 2 0000-11-16T07:32:49Z
counter negative_poolreq {pool=}
counter lease_conflicts
